schema/rdl_schema_0.1.json

```diff
@@ -1718,7 +1699,115 @@
         }
       }
     },
-    "vulnerability_f_math": {}
+    "vulnerability_f_math": {},
+    "Event": {
+      "title": "Event",
+      "Description": "A hazard event.",
+      "type": "object",
+      "required": [
+        "id",
+        "calculation_method",
+        "frequency_type"
+      ],
+      "properties": {
+        "id": {
+          "title": "Event identifier",
+          "type": "string",
+          "description": "The unique identifier for this event within the event_set.",
+          "minLength": 1
+        },
+        "disaster_identifier": {
+          "title": "Disaster identifier",
+          "type": "string",
+          "description": "The GLIDE number associated with this event disaster. A globally unique identification code for the observed or inferred hazard event. GLIDE numbers are available at [glidenumber.net](https://www.glidenumber.net/glide/public/search/search.jsp)."
+        },
+        "calculation_method": {
+          "title": "Model calculation method",
+          "type": "string",
+          "description": "The methodology used for the calculation of the event in the modelled scenario(s).",
+          "codelist": "data_calculation_type.csv",
+          "openCodelist": false
+        },
+        "frequency_type": {
+          "$ref": "#/$defs/common_frequency_type",
+          "description": "Frequency representation type"
+        },
+        "process_type": {
+          "$ref": "#/$defs/common_process_type",
```

Comment:
Replace with `title`, `description`, `codelist`, `openCodelist` and, if closed, `enum`.

Reply:
updated